app/components/collections/edit_link_component.rb

```diff
@@ -11,6 +11,15 @@ def initialize(collection_version:, anchor:, label:)
       @label = label
     end
 
+    sig { returns(String) }
+    def link_path
+      if @collection_version.first_draft?
+        edit_first_draft_collection_path(collection, anchor: anchor)
+      else
+        edit_collection_path(collection, anchor: anchor)
+      end
+    end
+
```

Comment:
this approach to ensuring we link to the correct edit collection page is sprinkled in other parts too but is not comprehensive and prone to missing a spot --- i also added a redirect in the collections controller to send you to the new route if needed, and I think that might make more sense that trying to catch all of the links everywhere